src/murmur/MurmurIce.cpp

```diff
@@ -57,6 +59,20 @@ static std::string iceString(const QString &s) {
 	return iceRemoveNul(u8(s));
 }
 
+/// iceStringBase64 converts the Ice-safe string in std::string
```

Comment:
Didn't jump into my eye here as much as on the other method comment, but this also names its name again